src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -4092,6 +4092,7 @@ function Test-ApplicationGatewayCRUDWithMutualAuthentication
 		$clientAuthConfig = Get-AzApplicationGatewayClientAuthConfiguration -SslProfile $sslProfile01
 		Assert-NotNull $clientAuthConfig
 		Assert-AreEqual $True $clientAuthConfig.VerifyClientCertIssuerDN
+		Assert-AreEqual "None" $clientAuthConfig.VerifyClientRevocation
```

Comment:
Please re-record this test case.

Reply:
session record is uploaded along with other updates, @wyunchi-ms , please help take review, Thanks!